core/App/navigators/NotificationStack.tsx

```diff
@@ -41,6 +43,11 @@ const NotificationStack: React.FC = () => {
         component={ProofRequestAttributeDetails}
         options={{ title: t('Screens.ProofRequestAttributeDetails') }}
       />
+      <Stack.Screen
+        name={Screens.CustomNotification}
+        component={customNotification.component}
+        options={{ title: t(customNotification.pageTitle as any) }}
```

Comment:
I might be able to cast it to something other than any, but I do need to cast it to something because the translation function expects it to be a key in the translation file